### ISSUE TITLE
Non-optimal regular expression: cyrillic ranges

### ISSUE THREAD

**Issue by putnik** [kalessil/phpinspectionsea#1288]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Plugin         | Php Inspections (EA Ultimate) v2.0.15 |
| Language level | PHP 7.1 |

Current behaviour
--
```php
preg_match('/[а-я]+/u', $field); // warning
preg_match('/[А-Я]+/u', $field); // warning
preg_match('/[a-z]+/u', $field); // no warning
preg_match('/[A-Z]+/u', $field); // no warning
```

Expected behaviour
--
Cyrillic letter ranges should not trigger a warning.

**Comment by kalessil** [comment]
@putnik: added test case, but can not reproduce. Can you supply a file with simplified code fragment?

**Comment by orklah** [comment]
Weird, I reproduce it with the code in example with Ulti V2.0.15 and last EAP

**Comment by kalessil** [comment]
What's the file encoding is?

**Comment by orklah** [comment]
UTF-8

Here's a capture
![Capture5](https://user-images.githubusercontent.com/9605520/55987658-4c88c580-5ca2-11e9-8f0c-215e0a113c9a.PNG)

**Comment by funivan** [comment]
Can reproduce:
Php inspections (EA Ultimate) v2.0.15

**Comment by kalessil** [comment]
Reproduced, fixed!
